### ISSUE TITLE
Hi

### ISSUE THREAD

**Issue by MichaelMure** [wagoodman/dive#88]
As the author of [git-bug](https://github.com/MichaelMure/git-bug), it's interesting to see that we made the same tech choice (go, gocui, cobra). Very neat and clean code, good job !

I'm just slightly annoyed by how good your `demo.gif` is. Could you share which tool/config you used to make it ?

Thanks a lot :-)

**Comment by pauloancheta** [comment]
[LiceCap](https://www.cockos.com/licecap/) does the same thing I think.

**Comment by ccinelli** [comment]
I use Giphy Capture

**Comment by wagoodman** [comment]
We indeed made similar choices, @MichaelMure --and thanks for noticing the clean[ish] code! As for the demo gif I ended up using https://github.com/phw/peek , they did a pretty solid job with the interface and minimal options. And since we're on useful tools, goreleaser (https://github.com/goreleaser/goreleaser) has also been pretty helpful too throughout this process. 

Git-bug is an awesome idea, I'm curious to see where it goes!
